### PR TITLE
feat: Added Sublime Text Online Code Editor in online-code-editors.json File in Frontend Category

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Fixes Issue
 Closes Issue No : #1383 

## Changes proposed
- Added `Sublime Text Online Code Editor in online-code-editors.json File in Frontend Category`
## Note to reviewers
- Sublime Text is a shareware text and source code editor available for Windows, macOS, and Linux. It natively supports many programming languages and markup languages. Users can customize it with themes and expand its functionality with plugins, typically community-built and maintained under free-software licenses.

### CODE REVIEW

**Comment by NarendraKoya999** [comment]
Hi @CBID2 . I have created PR. Please review on this as soon as possible

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -117,5 +117,12 @@
     "url": "https://www.interviewbit.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "Sublime Text",
+    "description": "Sublime Text is a shareware text and source code editor available for Windows, macOS, and Linux. It natively supports many programming languages and markup languages.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:123]:
```suggestion
    "description": "Sublime Text is a shareware text and source code editor available for Windows, macOS, and Linux. It supports many programming and markup languages.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just gave a suggestion on how to improve your pull request.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]

**Comment by rupali-codes** [comment]
@NarendraKoya999 please resolve merge conflicts

**Pull request merged**